docs/sources/next/javascript-api/k6-experimental/browser/browsercontext/addcookies.md

```diff
@@ -5,14 +5,20 @@ description: 'Clears context cookies.'
 
 # addCookies()
 
-Adds a list of [cookies](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/k6-experimental/browser/browsercontext/cookie) into the [BrowserContext](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/k6-experimental/browser/browsercontext/cookie). All pages within this [BrowserContext](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/k6-experimental/browser/browsercontext/cookie) will have these [cookies](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/k6-experimental/browser/browsercontext/cookie) set.
+Adds a list of [cookies](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/k6-experimental/browser/browsercontext/cookie) into the [browser context](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/k6-experimental/browser/browsercontext/). All pages within this [browser context](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/k6-experimental/browser/browsercontext/) will have these [cookies](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/k6-experimental/browser/browsercontext/cookie) set.
```

Comment:
I'm not sure about the `BrowserContext` to `browser context` change. I think it makes it easier for the user to see it used in one way, so that they can easily understand the relationship between all the components.

Why did you make the change?

Reply:
It was easier to read in a sentence. We've been using this pattern in other places (before this PR), too. I've created an issue to keep everything consistent: https://github.com/grafana/k6-docs/issues/1596.